bulls_and_cows/lib/api.ex

```diff
@@ -0,0 +1,34 @@
+defmodule BullsAndCows.API do
+  alias BullsAndCows
+  use GenServer
+
+  # Client
+
+  @doc """
+    Make number to guess to be picked, start the game
+  """
+  def start_link() do
```

Comment:
```suggestion
  def start_link do
```

In declarations, there are no parentheses. Don’t shoot the messenger :)